### PR TITLE
Update CI from Plugin Template

### PR DESCRIPTION
**Pull request by rochacbruno**
Depends on https://github.com/pulp/plugin_template/pull/470

No-Issue

### CODE REVIEW

File: .github/workflows/scripts/push_branch_and_tag_to_github.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@ ref_string=$(git show-ref --tags | grep refs/tags/$1)
 
 SHA=${ref_string:0:40}
 
-remote_repo=https://ansible:${RELEASE_TOKEN}@github.com/${GITHUB_REPOSITORY}.git
+remote_repo=https://pulpbot:${RELEASE_TOKEN}@github.com/${GITHUB_REPOSITORY}.git
```
Comment (fao89) [inline .github/workflows/scripts/push_branch_and_tag_to_github.sh:10]: I think you got the wrong user

File: template_config.yml [hunk 2]
```diff
@@ -67,7 +68,8 @@ pydocstyle: true
 pypi_username: null
 python_version: '3.8'
 redmine_project: null
-release_user: ansible
+release_email: pulp-infra@redhat.com
+release_user: pulpbot
```
Comment (fao89) [inline template_config.yml:72]: I believe you need to change this

File: .github/workflows/scripts/push_branch_and_tag_to_github.sh (hunk 1 above)
Comment (rochacbruno) [reply .github/workflows/scripts/push_branch_and_tag_to_github.sh:10]: We don´t have yet an `galaxybot` user, the token should be created for the specific user? maybe we need to create that user and add to the org.

Comment (fao89) [reply .github/workflows/scripts/push_branch_and_tag_to_github.sh:10]: not sure, as we have pulpbot we always used it, maybe other user will work, but I never tested

**Pull request closed without merging**